google-ads-examples/src/main/java/com/google/ads/googleads/examples/shoppingads/AddShoppingProductAd.java

```diff
@@ -237,11 +237,14 @@ private String addStandardShoppingCampaign(
             // the ads from immediately serving. Set to ENABLED once you've added
             // targeting and the ads are ready to serve.
             .setStatus(CampaignStatus.PAUSED)
-            // Sets the bidding strategy to Manual CPC (with eCPC enabled)
+            // Sets the bidding strategy to Manual CPC (with eCPC disabled). eCPC for standard
+            // Shopping campaigns is deprecated. If eCPC is set to true, Google Ads ignores the
+            // the setting and behaves as if the setting was false. See this blog post for more
+            // information: https://ads-developers.googleblog.com/2023/09/google-ads-shopping-campaign-enhanced.html
```

Comment:
Since URLs (including this one) almost always make a line exceed the 100 character standard, we usually put them in a separate comment line.

Reply:
Thanks for the info! Done moving it to a separate line.